content/en/monitors/downtimes/_index.md

```diff
@@ -86,7 +86,7 @@ There are a few limitations that are **not supported** which include:
 
 * More than two levels of nesting, such as `team:app AND (service:auth OR (service:graphics-writer AND (env:prod OR (type:metric AND status:ok))))`, are not supported. At most, Downtimes accept two levels of nesting. Use separate Downtimes instead to break down the logic.
 * Negation is only supported for key/value pairs and tags with `OR`. For example, `-key:value` and `-key(A OR B)`. Scopes such as `-service:(A AND B)`, `service:(-A OR -B)`, or `-service(A B)` are not supported.
-* Top level ORs are not supported, for example, `service:A OR host:X`. This requires two separate Downtimes.
+* Top level ORs are not supported. For example, `service:A OR service:B` will work but `service:A OR host:X` will not; this requires two separate Downtimes.
```

Comment:
```suggestion
* Top level ORs are not supported. For example, `service:A OR service:B` is valid, but `service:A OR host:X` does not work. An OR between two different top level tags requires two separate Downtimes.
```

Reply:
Thank you for improving my improvement! This is much clearer